src/runtime-rs/crates/hypervisor/src/device/driver/virtio_vsock.rs

```diff
@@ -118,15 +118,21 @@ const CID_RETRY_COUNT: u32 = 50;
 
 impl VsockDevice {
     pub async fn new(id: String) -> Result<Self> {
-        let (guest_cid, _vhost_fd) = generate_vhost_vsock_cid()
-            .await
-            .context("generate vhost vsock cid failed")?;
-
         Ok(Self {
             id,
-            config: VsockConfig { guest_cid },
+            config: VsockConfig {
+                guest_cid: libc::VMADDR_CID_ANY,
+            },
         })
     }
+
+    pub async fn init_config(&mut self) -> Result<File> {
```

Comment:
seems like this function is not used anywhere, will the actual usage of this will be introduced later?

Reply:
yeah, coming soon from https://github.com/kata-containers/kata-containers/pull/8185 😉